pam/internal/adapter/newpasswordmodel.go

```diff
@@ -91,19 +93,28 @@ func (m newPasswordModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, nil
 
 		case "tab":
+			m.errorMsg = ""
+			if m.focusIndex == 0 && m.passwordEntries[0].Value() != "" {
```

Comment:
on `m.passwordEntries[0].Value() != ""`, indeed pwquality includes such check, but is this a case we may want to leave brokers to accept that?

Reply:
I don't understand exactly what you meant here.

Reply:
libpwquality has a check for empty password:

https://github.com/libpwquality/libpwquality/blob/80deb29da767961676fedd87dbf264acc438a720/src/check.c#L693-L695

So I was wondering if here it was wanted to skip the check in case of empty password, to delegate this to the broker instead

Reply:
Not exactly (even though the broker does not allow empty password anyway). This is mainly to avoid triggering the check on a empty password (in a case where the user didn't type anything yet and pressed tab without meaning to)

Reply:
Mh... I see. So maybe in case of empty ones we should still trigger it on final `Enter`?

Reply:
In the sense, if we want to rely this to brokers the whole PR here doesn't make sense, but if we want to provide a minimum filter, we should probably do it also for the empty case.

However, I don't see a problem showing an error in the empty case too, IMHO.